### ISSUE TITLE
Explicitly document the lack of an update path

### ISSUE THREAD

**Issue by phenaproxima** [phenaproxima/starshot-prototype#78]
Starshot has no update paths, crossgrade paths, or migration paths for anything. That is not in its scope or its mission.

When you start a project with Starshot, you get the best starting point we can offer, at that time. Best practices may evolve, but Starshot is NOT responsible for keeping you up-to-date. Modules can provide update paths for their own data, but that's not something Starshot will do. Distributions have traditionally tried to do this by various means, but Starshot is not a distribution and shouldn't be expected to act like one.

I think that we should explicitly and clearly document this decision and the rationale behind it, so that we have somewhere to point people when this question comes up.

**Comment by simesy** [comment]
I agree and it's mostly about awareness, so just an agreement form/layover?

**Comment by simesy** [comment]
... or just promote the Upgrade Manager.
